great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -461,6 +463,58 @@ def init_expectation_configuration_builder(
     return expectation_configuration_builder
 
 
+def resolve_evaluation_dependencies(
+    parameter_builder: "ParameterBuilder",  # noqa: F821
+    parameter_container: ParameterContainer,
+    domain: Domain,
+    variables: Optional[ParameterContainer] = None,
+    parameters: Optional[Dict[str, ParameterContainer]] = None,
+) -> None:
```

Comment:
Could we add a docstring here?

Reply:
@cdkini I will add it (in the next commit).

Reply:
@cdkini I added a docstring.  Thanks!